### PR TITLE
Add methods for working with contacts.

### PR DESCRIPTION
**Pull request by CheshireCaat**
Added custom constructor for TLVector. This makes it easy to create a TLVector object from other collections.

Added methods for working with contacts:
- Import contact list
- Delete contact list
- Delete one contact

### CODE REVIEW

File: TeleSharp.TL/TLVector.cs [hunk 1]
```diff
@@ -11,7 +11,17 @@ namespace TeleSharp.TL
...
+
+        public TLVector(IEnumerable<T> collection)
+        {
+            lists = new List<T>(collection);
+        }
+
+        public TLVector()
+        {
+            lists = new List<T>();
+        }
```
Comment (knocte) [inline TeleSharp.TL/TLVector.cs]: thanks for your contribution! can you remove this from the PR? seems unrelated with the problem at hand

Comment (CheshireCaat) [reply TeleSharp.TL/TLVector.cs]: Okay, i'll solve it a bit later

Comment (knocte) [reply TeleSharp.TL/TLVector.cs]: wait, I think I'm mistaken, sorry, it is related

File: TLSharp.Core/TelegramClient.cs [hunk 2]
```diff
@@ -234,12 +234,42 @@ public async Task<T> SendRequestAsync<T>(TLMethod methodToExecute)
...
+            return await SendRequestAsync<TLLink>(req);
+        }
+
         public async Task<TLContacts> GetContactsAsync()
         {
             if (!IsUserAuthorized())
                 throw new InvalidOperationException("Authorize user first!");
 
-            var req = new TLRequestGetContacts() { Hash = "" };
+            var req = new TLRequestGetContacts {Hash = ""};
```
Comment (knocte) [inline TLSharp.Core/TelegramClient.cs]: what is not related is this change

File: TeleSharp.TL/TLVector.cs (hunk 1 above)
Comment (CheshireCaat) [reply TeleSharp.TL/TLVector.cs]: Well, I thought that you want to leave the default constructor empty and leave the list initialization as it was before. If there are no problems, this is very good)

File: TLSharp.Core/TelegramClient.cs (hunk 2 above)
Comment (CheshireCaat) [reply TLSharp.Core/TelegramClient.cs]: Fixed
